packages/sitecore-jss-nextjs/src/context/context.ts

```diff
@@ -137,12 +137,17 @@ export class Context<SDKModules extends SDKModulesType> {
    * @returns {void}
    */
   protected initSDK<T extends keyof SDKModules>(name: T): void {
-    this.sdkPromises[name] = new Promise((resolve) => {
-      this.props.sdks[name].init(this).then(() => {
-        this.sdks[name] = this.props.sdks[name].sdk;
-
-        resolve(this.sdks[name]);
-      });
+    this.sdkPromises[name] = new Promise((resolve, reject) => {
+      this.props.sdks[name]
+        .init(this)
+        .then(() => {
+          this.sdks[name] = this.props.sdks[name].sdk;
+          resolve(this.sdks[name]);
+        })
+        .catch((e) => {
+          console.log(`Initialization for SDK ${name.toString()} skipped. Reason: \n ${e}`);
+          reject(e);
+        });
```

Comment:
Do we need to catch and reject explicitly here or can we just let it bubble up? If we are doing it just to be able to console.log, don't know if that is necessary.

Reply:
Removed the log here - the error will now only show when attempting a getSDK on rejecting SDK